src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/GamingBacklogApiApplication.kt

```diff
@@ -2,10 +2,30 @@ package com.gamingbacklog.api.gamingbacklogapi
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.CorsRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 
 @SpringBootApplication
 class GamingBacklogApiApplication
 
 fun main(args: Array<String>) {
 	runApplication<GamingBacklogApiApplication>(*args)
 }
+
+@Configuration
+class CorsConfig {
+
+  @Bean
+  fun corsConfigurer(): WebMvcConfigurer {
```

Comment:
Do we have anything that can test this? If everything looks okay when you test locally then that's probably enough for now but curious if there's any way we can systematically make sure this is working

Reply:
It did work locally. I'm not sure if there is a great way to systematically test to be honest